app/frontend/users/UserEdit.res

```diff
@@ -421,6 +443,8 @@ let make = (
     deletingAccount: false,
     avatarUploadError: None,
     dirty: false,
+    newPasswordAnalysis: None,
+    schoolName: schoolName,
```

Comment:
This is something that the user will never change and need not be a state attribute. Pass the prop around wherever needed